GliaWidgets/Public/Glia/Glia.swift

```diff
@@ -100,6 +102,41 @@ public class Glia {
                     callback()
             }
         }
+
+        var viewFactory = ViewFactory(
+            with: Theme(),
+            messageRenderer: messageRenderer,
+            environment: .init(
+                data: environment.data,
+                uuid: environment.uuid,
+                gcd: environment.gcd,
+                imageViewCache: environment.imageViewCache,
+                timerProviding: environment.timerProviding,
+                uiApplication: environment.uiApplication
+            )
+        )
+
+        callVisualizerCoordinator = .init(
+            viewFactory: viewFactory,
+            presenter: .topViewController(application: .live)
```

Comment:
think we should pass application parameter explicitly from Glia.Environment. So that resulting line would be:
`presenter: .topViewController(application: environment.uiApplication)`

Reply:
fixed - 1b0731d2195a403fc3f09603a2a7a6a9b046ae07